src/barril/units/_quantity.py

```diff
@@ -4,11 +4,16 @@
 
 from barril._foundation.odict import odict
 from barril.units.exceptions import QuantityValidationError
-from barril.units.unit_database import UnitDatabase, UnitsError
+from barril.units.unit_database import (
+    InvalidUnitError,
+    UnitDatabase,
+    UnitsError,
+    fix_unit_if_is_legacy,
```

Comment:
The name convention on the lib is capitalized camel case but it is fine to let it the way it is.